blazor/gantt-chart/clipboard.md

```diff
@@ -0,0 +1,495 @@
+        }
+        else if (Args.Value == "Child")
+        {
+            CopyType = CopyHierarchyType.Child;
+        }
+        else if (Args.Value == "Both")
+        {
+            CopyType = CopyHierarchyType.Both;
+        }
+        else if(Args.Value == "None")
+        {
+            CopyType = CopyHierarchyType.None;
+        }
+    }
+
+    public async void ToolbarClick(Syncfusion.Blazor.Navigations.ClickEventArgs args)
+    {
+        var SelectedRecords = await this.GanttObject.GetSelectedRecordsAsync();
+        if (SelectedRecords.Count() > 0)
+        {
+            var WithHeader = false;
+            if (args.Item.Id == "copyHeader")
+            {
+                WithHeader = true;
+            }
+            await this.GanttObject.CopyAsync(WithHeader);
+        }
+    }
+    public class TaskData
+    {
+        public int TaskId { get; set; }
+        public string TaskName { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public string Duration { get; set; }
+        public int Progress { get; set; }
+        public string Predecessor { get; set; }
+        public int? ParentId { get; set; }
+    }
+
+    public static List<TaskData> GetTaskCollection()
+    {
+        List<TaskData> Tasks = new List<TaskData>()
+        {
+            new TaskData() { TaskId = 1, TaskName = "Project initiation", StartDate = new DateTime(2019, 04, 05), EndDate = new DateTime(2019, 04, 21), },
+            new TaskData() { TaskId = 2, TaskName = "Identify Site location", StartDate = new DateTime(2019, 04, 05), Duration = "0", Progress = 30, ParentId = 1 },
+            new TaskData() { TaskId = 3, TaskName = "Perform soil test", StartDate = new DateTime(2019, 04, 05), Duration = "4", Progress = 40, Predecessor = "2", ParentId = 1 },
+            new TaskData() { TaskId = 4, TaskName = "Soil test approval", StartDate = new DateTime(2019, 04, 05), Duration = "0", Progress = 30, Predecessor = "3", ParentId = 1 },
+            new TaskData() { TaskId = 5, TaskName = "Project estimation", StartDate = new DateTime(2019, 04, 06), EndDate = new DateTime(2019, 04, 21) },
+            new TaskData() { TaskId = 6, TaskName = "Develop floor plan for estimation", StartDate = new DateTime(2019, 04, 06), Duration = "3", Progress = 30, Predecessor = "4" , ParentId = 5 },
+            new TaskData() { TaskId = 7, TaskName = "List materials", StartDate = new DateTime(2019, 04, 06), Duration = "3", Progress = 40, Predecessor = "6" , ParentId = 5 },
+            new TaskData() { TaskId = 8, TaskName = "Estimation approval", StartDate = new DateTime(2019, 04, 06), Duration = "0", Progress = 30, Predecessor = "7" , ParentId = 5 },
+        };
+        return Tasks;
+    }
+}
+```
+
+## Paste
+
+The content of a row/cells can be copied by selecting the rows/cells and pressing <kbd>Ctrl + C</kbd> shortcut key to copy data and paste it by pressing <kbd>Ctrl + V</kbd> shortcut key.
```

Comment:
Do we support paste functionality?

Reply:
Provided a work around solution.
Copied and pasted a row using AddRecordAsync method.
Copied a cell and pasted by using UpdateRecordByIDAsync method.